### ISSUE TITLE
web-console markup & javascript is outside the <html> element of the page

### ISSUE THREAD

**Issue by sarahmei** [rails/web-console#125]
...and it confuses Capybara.

When I have web-console installed, and my Capybara test hits a Rails error page, I get this super confusing error message:

```
  1) index does the thing
     Failure/Error: expect(page).to have_content('a thing')
     Capybara::Ambiguous:
       Ambiguous match, found 2 elements matching xpath "/html"
     # /Users/steampunk/.rvm/gems/ruby-2.2.1/gems/capybara-2.4.4/lib/capybara/node/finders.rb:38:in `block in find'
     # /Users/steampunk/.rvm/gems/ruby-2.2.1/gems/capybara-2.4.4/lib/capybara/node/base.rb:80:in `synchronize'
     # /Users/steampunk/.rvm/gems/ruby-2.2.1/gems/capybara-2.4.4/lib/capybara/node/finders.rb:30:in `find'
     # /Users/steampunk/.rvm/gems/ruby-2.2.1/gems/capybara-2.4.4/lib/capybara/node/document.rb:23:in `text'
     # /Users/steampunk/.rvm/gems/ruby-2.2.1/gems/capybara-2.4.4/lib/capybara/queries/text_query.rb:26:in `resolve_for'
     # /Users/steampunk/.rvm/gems/ruby-2.2.1/gems/capybara-2.4.4/lib/capybara/node/matchers.rb:454:in `block in assert_text'
     # /Users/steampunk/.rvm/gems/ruby-2.2.1/gems/capybara-2.4.4/lib/capybara/node/base.rb:84:in `synchronize'
     # /Users/steampunk/.rvm/gems/ruby-2.2.1/gems/capybara-2.4.4/lib/capybara/node/matchers.rb:453:in `assert_text'
     # /Users/steampunk/.rvm/gems/ruby-2.2.1/gems/capybara-2.4.4/lib/capybara/session.rb:676:in `block (2 levels) in <class:Session>'
     # /Users/steampunk/.rvm/gems/ruby-2.2.1/gems/capybara-2.4.4/lib/capybara/rspec/matchers.rb:64:in `matches?'
     # ./spec/features/index_spec.rb:6:in `block (2 levels) in <top (required)>'
```

This error occurs because web-console's HTML elements (including script tags) are placed at the bottom of the file, outside the `</html>`. As a result, Capybara thinks there are two webpages in the file, and gives up.

Is it possible to move those elements inside the `html` tag?

**Comment by gsamokovarov** [comment]
This is something we did from the beginning as the easiest thing to get us going. Browsers usually don't complain and take whatever they have to work with, however I think this is a legitimate concern. I think we can put some work into injecting the markup at a more appropriate place.

Just curios, did you forgot the `console` invocation during the test run or was it intended to be there?

**Comment by sarahmei** [comment]
I didn't invoke the console at all, as far as I know. All I have to do is include the web-console gem in my Gemfile, and then the elements & JS get injected into the error pages. I'm assuming this is automatic and on-purpose. :)

**Comment by chancancode** [comment]
Hm, do we intend for web-console to run in test mode? Seems like it won't be useful there (and as it turned out, can cause problems). Generally, the test environment should be closer to production than development, so it probably shouldn't run in test mode?

**Comment by gsamokovarov** [comment]
@chancancode we never intended it to be used in test mode, but we don't enforce anything during runtime anymore.

**Comment by gsamokovarov** [comment]
This has been fixed by #162.
